client/app/components/visualizations/visualizationComponents.jsx

```diff
@@ -11,6 +12,7 @@ function wrapComponentWithSettings(WrappedComponent) {
   return function VisualizationComponent(props) {
     updateVisualizationsSettings({
       HelpTriggerComponent: HelpTrigger,
+      LinkComponent: Link,
```

Comment:
Is this to support ability to handle correctly internal links in tables?

Reply:
Yes, exactly